app/src/main/java/io/github/pulquero/rotordroid/MainActivity.java

```diff
@@ -35,10 +35,10 @@
 import java.util.concurrent.TimeUnit;
 
 public class MainActivity extends AppCompatActivity {
-    private static final int MIN_FREQ = 5645;
-    private static final int MAX_FREQ = 5945;
+    private static int MIN_FREQ = 5600;
```

Comment:
These variables should probably not be static any more given they are mutable within an instance.